src/DataViews/LiveFunctionsDataView.cpp

```diff
@@ -96,14 +112,20 @@ std::string LiveFunctionsDataView::GetValue(int row, int column) {
 
   const uint64_t scope_id = GetScopeId(row);
   const ScopeStats& stats = app_->GetCaptureData().GetScopeStatsOrDefault(scope_id);
+  const orbit_client_data::ScopeInfo& scope_info = GetScopeInfo(scope_id);
 
   const FunctionInfo* function = GetFunctionInfoFromRow(row);
   switch (column) {
-    case kColumnSelected:
-      return function == nullptr ? ""
-                                 : FunctionsDataView::BuildSelectedColumnsString(app_, *function);
+    case kColumnType: {
+      const std::string prefix =
+          function == nullptr
+              ? ""
+              : FunctionsDataView::BuildSelectedAndFrametrackIconsString(app_, *function);
+
+      return absl::StrCat(prefix, prefix.empty() ? "" : " ", BuildTypeColumnsString(scope_info));
```

Comment:
I agree with putting the type first (e.g., "D F" instead of "F D"). But I think we can still improve the separation between the two parts, for example "D [H]" instead of "D H".

Reply:
+1 for "D [H]"

Reply:
Great idea.